src/pages/SidebarPage.tsx

```diff
@@ -0,0 +1,229 @@
+          Dashboard
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiViewBoards}>
+          Kanban
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiInbox}>
+          Inbox
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiUser}>
+          Users
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiShoppingBag}>
+          Products
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiArrowSmRight}>
+          Sign In
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiTable}>
+          Sign Up
+        </Sidebar.Item>
+      </Sidebar.ItemGroup>
+      <Sidebar.ItemGroup>
+        <Sidebar.Item href="#" icon={HiChartPie}>
+          Upgrade to Pro
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiViewBoards}>
+          Documentation
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={BiBuoy}>
+          Help
+        </Sidebar.Item>
+      </Sidebar.ItemGroup>
+    </Sidebar.Items>
+  </Sidebar>
+);
+
+export const CTASidebarComponent = () => (
+  <Sidebar collapsed={false}>
+    <Sidebar.Items>
+      <Sidebar.ItemGroup>
+        <Sidebar.Item href="#" icon={HiChartPie}>
+          Dashboard
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiViewBoards}>
+          Kanban
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiInbox}>
+          Inbox
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiUser}>
+          Users
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiShoppingBag}>
+          Products
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiArrowSmRight}>
+          Sign In
+        </Sidebar.Item>
+        <Sidebar.Item href="#" icon={HiTable}>
+          Sign Up
+        </Sidebar.Item>
+      </Sidebar.ItemGroup>
+    </Sidebar.Items>
+    <Sidebar.CTA>
+      <div className="mb-3 flex items-center">
+        <span className="mr-2 rounded bg-orange-100 px-2.5 py-0.5 text-sm font-semibold text-orange-800 dark:bg-orange-200 dark:text-orange-900">
+          Beta
+        </span>
+        <button
+          type="button"
+          className="-mx-1.5 -my-1.5 ml-auto inline-flex h-6 w-6 rounded-lg bg-blue-50 p-1 text-blue-900 hover:bg-blue-200 focus:ring-2 focus:ring-blue-400 dark:bg-blue-900 dark:text-blue-400 dark:hover:bg-blue-800"
+          data-collapse-toggle="dropdown-cta"
+          aria-label="Close"
+        >
+          <span className="sr-only">Close</span>
+          <svg className="h-4 w-4" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+            <path
+              fillRule="evenodd"
+              d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z"
+              clipRule="evenodd"
+            ></path>
+          </svg>
+        </button>
```

Comment:
For this button, can't we use our homemade `Button` component and also an SVG component from `react-icons`?

Reply:
Thanks, I bet I copied this from the `flowbite` example and forgot to replace those with the React components. Let me do that so these examples are more accurate.